ChaosMod/Effects/db/Player/PlayerTpController.cpp

```diff
@@ -291,10 +303,19 @@ static void OnStartFakeTp()
 		SET_ENTITY_INVINCIBLE(playerVeh, true);
 	}
 
+	int currentWanted = GET_PLAYER_WANTED_LEVEL(player);
+	int wanted = GetFakeWantedLevel(randLocation.first);
+	if (wanted == 0 || wanted < currentWanted)
+	{
+		wanted = currentWanted;
+	}
+
 	SET_PLAYER_WANTED_LEVEL(player, 0, false);
 	SET_PLAYER_WANTED_LEVEL_NOW(player, false);
 	SET_MAX_WANTED_LEVEL(0);
 
+	SET_FAKE_WANTED_LEVEL(wanted);
```

Comment:
Noticing in testing that the fake wanted level has a different effect than the real one... No police are dispatched so the effect might not be as convincing this way. Could adjust the real wanted level alternatively

Reply:
I've already commented on the original commit that added the new script-pause protections, but pongo must've missed it.

With the script-pause being a thing, it should no longer be possible for the mission to fail due to temporary real wanted level. So the real wanted level should never be set to 0 to begin with, it should just be re-set to whatever it was before teleporting.

Reply:
Good point! Could just ensure that any existing fake or real wanted level is persisted after the effect. Just set a real wanted level, and reset to original values after the fake TP is revealed

Reply:
The wanted level has to be set before the script-pause though, otherwise it wont be shown -> would potentially cause mission fails. this is "good enough" imo :)